exercises/rectangles/src/test/java/RectangleCounterTest.java

```diff
@@ -14,12 +14,20 @@ public void setUp() {
     }
 
     @Test
-    public void testInputWithNoColumnsContainsNoRectangles() {
+    public void testInputWithNoRowsContainsNoRectangles() {
         String[] inputGrid = new String[]{};
 
         assertEquals(0, rectangleCounter.countRectangles(inputGrid));
     }
 
+    @Ignore("Remove to run test")
+    @Test
+    public void testInputWithNoColumnsContainsNoRectangles() {
+    	String[] inputGrid = new String[]{""};
+    	
+    	assertEquals(0, rectangleCounter.countRectangles(inputGrid));
```

Comment:
I think the indentation might be a bit off here? It should be indented with four spaces, same as the other tests 🙂